### PR TITLE
test: add test cases for IssueUtil

### PR DESCRIPTION
**Pull request by Mini256**
add test for IssueUtil (#32 )

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
     "build:watch": "tsc && (tsc -w --preserveWatchOutput & nodemon)",
     "dev": "npm run build:watch",
     "start": "probot run ./lib/index.js",
-    "lint": "prettier --check 'src/**/*.ts' 'test/**/*.ts' *.md package.json tsconfig.json",
-    "lint:fix": "prettier --write 'src/**/*.ts' 'test/**/*.ts' *.md package.json tsconfig.json",
+    "lint": "prettier --check \"src/**/*.ts\" \"test/**/*.ts\" *.md package.json tsconfig.json",
```
Comment (hi-rustin) [inline package.json:21]: I think maybe you should rebase your PR.

**Review by hi-rustin — CHANGES_REQUESTED** [review]

File: test/services/utils/IssueUtil.test.ts [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+} from "../../../src/services/utils/IssueUtil";
+import {
+  CHALLENGE_PROGRAM_LABEL,
+  HELP_WANTED_LABEL,
+} from "../../../src/commands/labels";
+
+describe("Issue Util", () => {
+  test("issue have sig label", () => {
+    expect(
+      findSigLabel([
```
Comment (hi-rustin) [inline test/services/utils/IssueUtil.test.ts:17]: I think that after calling the function, storing the return value through a variable and then asserting may be more readable.

**Review by hi-rustin — CHANGES_REQUESTED** [review]

File: test/services/utils/IssueUtil.test.ts [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+  isClosed,
+  findMentorAndScore,
+} from "../../../src/services/utils/IssueUtil";
+import {
+  CHALLENGE_PROGRAM_LABEL,
+  HELP_WANTED_LABEL,
+} from "../../../src/commands/labels";
+
+describe("Issue Util", () => {
+  test("issue have sig label", () => {
```
Comment (hi-rustin) [inline test/services/utils/IssueUtil.test.ts:15]: It think the description should be `test("sig label can be found")`.

It is best to check if your other test descriptions are appropriate.

**Review by hi-rustin — CHANGES_REQUESTED** [review]

File: test/services/utils/IssueUtil.test.ts (hunk 3 above)
Comment (Mini256) [reply test/services/utils/IssueUtil.test.ts:15]: ok

Comment (hi-rustin) [reply test/services/utils/IssueUtil.test.ts]: It's best to use `can` or `could` in all descriptions.

File: .gitignore [hunk 4]
```diff
@@ -111,3 +111,4 @@ dist
 # IDEA
 .idea
 /docs/_book/
+bot-logs/ti-challenge-bot.log.0
```
Comment (hi-rustin) [inline .gitignore:114]: Could you please ignore the whole dir?

**Review by hi-rustin — CHANGES_REQUESTED** [review]

**Review by hi-rustin — COMMENTED** [review]
rest LGTM

File: .gitignore [hunk 5]
```diff
@@ -111,3 +111,4 @@ dist
 # IDEA
 .idea
 /docs/_book/
+bot-logs/
```
Comment (hi-rustin) [inline .gitignore:114]: Better add a new line and comment what it is.

**Comment by hi-rustin** [comment]
@Mini256  Good job 👏

**Pull request merged**